corehq/apps/export/models/new.py

```diff
@@ -671,6 +676,7 @@ class ExportInstance(BlobMixin, Document):
     # daily saved export fields:
     last_updated = DateTimeProperty()
     last_accessed = DateTimeProperty()
+    last_duration = IntegerProperty()
```

Comment:
@esoergel @snopoke 
Wanted to check if its possible and correct to make this a TimeProperty instead? I was trying that initially and had challenge assigning value to it in the expected format. I had an integer value.